stl/inc/yvals.h

```diff
@@ -242,12 +242,16 @@ _EMIT_STL_ERROR(STL1008, "_STL_CALL_ABORT_INSTEAD_OF_INVALID_PARAMETER has been
 //     a non-void function, etc.), but it will not attempt to replace undefined behavior with implementation-defined
 //     behavior. (For example, we will not transform `pop_back()` of an empty `vector` to be a no-op.)
 #ifndef _MSVC_STL_DOOM_FUNCTION
-#ifdef _MSVC_STL_USE_ABORT_AS_DOOM_FUNCTION
+#ifdef _MSVC_STL_USE_ABORT_AS_DOOM_FUNCTION // The user wants to use abort():
 #define _MSVC_STL_DOOM_FUNCTION(mesg) _CSTD abort()
-#else // ^^^ defined(_MSVC_STL_USE_ABORT_AS_DOOM_FUNCTION) / !defined(_MSVC_STL_USE_ABORT_AS_DOOM_FUNCTION) vvv
-// TRANSITION, GH-4858: after dropping Win7 support, we can directly call __fastfail(FAST_FAIL_INVALID_ARG).
+#elif defined(__clang__) // Use the Clang intrinsic:

```

Comment:
Wanted to ask for `#elifdef` but then recalled about the variety of targets, so perhaps no `#elifdef` in headers for now

Reply:
Yeah, we have to barely tolerate non-compiler tools like rc.exe, Qt MoC, etc. I noticed your use of elifdef in `stl/src` and didn't complain because that's MSVC-only, but in headers I don't want to try.